paramak/parametric_components/hexagon_pin.py

```diff
@@ -0,0 +1,96 @@
+
+import math
+from typing import Optional, Tuple
+
+from paramak import ExtrudeStraightShape
+
+
+class HexagonPin(ExtrudeStraightShape):
+    """Creates an extruded hexagon by a provided distance about a center point.
+
+    Args:
+        length_of_side: the vertical (z axis) height of the coil (cm).
+        width: the horizontal (x axis) width of the coil (cm).
```

Comment:
I think width should be distance so that it's consisten with the init function

Reply:
thanks for that, issue has been fixed